README.md

```diff
@@ -43,58 +43,61 @@ privileges and forfeiture of registration fee.
 
 ## Client Quick Start Installation
 
-- It is recommended to have at least 2 GB of available system memory and 25 GB of free disk space to run the DEX client. For the most secure setup, you can also install in a fresh virtual machine.
+The DEX client can be installed in one of three ways:
 
-You can use the
-[**dcrinstall**](https://github.com/decred/decred-release/releases) tool to
-install everything you need. **dcrinstall** will guide you through installation
-of all needed software and help set up your Decred wallet.
+1. [Install Decrediton](https://docs.decred.org/wallets/decrediton/decrediton-setup/) and go to the DEX tab.
+2. Install the standalone DEX client using `dcrinstall` as described below.
+3. Build the standalone client [from source](#advanced-client-installation).
 
-`./dcrinstall --dcrdex`
+If available for the latest release, the
+[**dcrinstall**](https://docs.decred.org/wallets/cli/cli-installation/) tool
+will install everything you need, and help set up your Decred wallet. Just run
+dcrinstall from the command line with `--dcrdex` appended to the command shown
+on the linked page for your operating system. Otherwise see the latest DCRDEX
+releases [on Github](https://github.com/decred/dcrdex/releases) for source code
+or pre-compiled standalone packages.
```

Comment:
I think this last sentence will be my preferred installation method that I'd recommend for 0.5.  I will just upload the archives generated the pkg.sh script to the github release when it's published.  I have a hard time seeing `dcrinstall` being a good install method for dcrdex given neither dcrd, dcrwallet, or bitcoind are needed anymore, and there's no point in scaring users with one more CLI tool.